src/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+set(libbinio_la_SOURCES binio.cpp binfile.cpp binwrap.cpp binstr.cpp)
+configure_file(binio.h.in ${CMAKE_SOURCE_DIR}/src/binio.h)
+set(pkginclude_HEADERS binio.h binfile.h binwrap.h binstr.h)
+add_library(libbinio ${libbinio_la_SOURCES} ${pkginclude_HEADERS})
+target_link_options(libbinio PRIVATE -version-info 1:0:0)
```

Comment:
Where is the `-version-info` value coming from?  I can't see this specified in the existing build system, and it would be good if both of them produced the same output library.

Reply:
oops! That was a bad copy-paste from another project